### ISSUE TITLE
Contribution to the OpenModelica code base

### ISSUE THREAD

**Issue by casella** [PR-DC/PRDC_OMOctave#1]
@mpetrasinovic, would you be interested at providing your solution directly to the OpenModelica codebase?

We can't do that currently because your code is GPL'd, and we also release OMC under the OSMC-PL license. If you contribute yourself, you'd be transferrring copyright to the non-profit Open Source Modelica Consortium, that will release it under dual license (GPL and OSMC-PL).

Thanks!

**Comment by mpetrasinovic** [comment]
@casella Yes, we would like to do that. Can you help us, how we should do it?
